doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The number of RPCs is configured by the experimental flag, `soak_iterations`.
+
+This test is configurable via a few different command line flags:
+
+* `soak_iterations`: controls the number of RPCs to perform.
+
+* `soak_max_failures`: an inclusive upper limit on the number of RPC failures
+  that should be tolerated (i.e. after which the test process should
+  still exit 0). A failure is considered to be either a non-OK status or an RPC
+  whose latency exceeds `soak_per_iteration_max_acceptable_latency_ms`.
+
+* `soak_per_iteration_max_acceptable_latency_ms`: an inclusive upper limit
+  on the latency of a single RPC in order for that RPC to be considered
+  successful.
+
+* `soak_overall_timeout_seconds`: the overall number of seconds after which
+  the test should stop and fail if `soak_iterations` have not yet been
+  completed.
+
+The following suggestions are optional but encouraged to improve debuggability:
+
+* Implementations should avoid setting RPC deadlines and should instead
```

Comment:
I don't understand this. Why shouldn't an implementation set the deadline based on soak_per_iteration_max_acceptable_latency_ms?

Reply:
This just provides more data for debugging in case of failure.

E.g., if we set the deadline and then exceeded, it's not clear if the RPC would have taken a few more milliseconds or another minute.